layouts/jobs/list.html

```diff
@@ -350,8 +407,10 @@ <h5 class="modal-title" id="descriptionModalLabel${ind}">${job.title}</h5>
     // Call the function to fetch, parse and display jobs
     getDataFromUrl(
       decodeURIComponent("https%3A%2F%2Fblobserver.dc.scilifelab.se%2Fblob%2Fdata_platform_jobs.json"),
-      addJobs,
-      enableFiltering
+      function(jobsData) {
+        addJobs(jobsData);
+        enableFiltering();
+      }
```

Comment:
As mentioned, the idea is to stop using manually produced data in blob server and use direct approach. Also the new calling could be something like 

```
async function getDataFromUrl(url) { ....... }

async function addJobs (dataArray) { ....... }

function enableFiltering() { ........ }

document.addEventListener("DOMContentLoaded", async () => {
    const availableJobs = await getDataFromUrl('https://www.scilifelab.se/wp-json/wp/v2/career?orderby=archive_date&per_page=50');
    await addJobs(availableJobs);
    enableFiltering();
})
```